ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-main-presubmits.yaml

```diff
@@ -1670,6 +1674,7 @@ presubmits:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-machine-config-operator-main-e2e-gcp-op
+    optional: true
```

Comment:
Hmm, something seems weird, this test (plus some other ones in this list) should be required.

I'm auditing the tests now and will open a follow up.